<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOffsetModel</class>
 <widget class="QDialog" name="EditOffsetModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Offset Model</string>
  </property>
  <widget class="QPushButton" name="valid_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinbox_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.990000000000009</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>33</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>73</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinbox_y">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.990000000000009</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_z">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>113</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinbox_z">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.990000000000009</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
